src/UserService.Business/Commands/User/EditUserCommand.cs

```diff
@@ -146,32 +111,23 @@ public async Task<OperationResultResponse<bool>> ExecuteAsync(Guid userId, JsonP
         await _accessValidator.HasRightsAsync(Rights.AddEditRemoveUsers) ||
         (userId == requestSenderId
         && roleOperation == null
-        && officeOperation == null
         && isActiveOperation == null)))
       {
         _responseCreater.CreateFailureResponse<bool>(HttpStatusCode.Forbidden);
       }
 
       OperationResultResponse<bool> response = new();
 
-      List<string> errors = new List<string>();
+      var errors = new List<string>();
```

Comment:
why do you rewrite to var?

Reply:
because here the type of the variable is explicit